scripts/run_backend_tests.py

```diff
@@ -475,6 +469,13 @@ def main(args: Optional[List[str]] = None) -> None:
     """Run the tests."""
     parsed_args = _PARSER.parse_args(args=args)
 
+    # Installing third-party Oppia dependencies is unnecessary
+    # when running the test with the --test_target flag. It is done
+    # for performance improvement.
+    if not parsed_args.test_target:
+        from . import install_third_party_libs
+        install_third_party_libs.main()
```

Comment:
Can we just have the `install_third_party_libs.main()` inside `if case`?

Also, should we introduce a separate flag `skip_install` to allow users to skip installation?

Reply:
Done!
Hey, do we need to implement a test for this change? If so, which testing module to use `unittest` or `pytest`. I see both of them are used in our codebase.

Reply:
THanks for the changes, @shricodev! It looks better annd I've left one comment, PTAL!

I think we only use `unittest` throughout the project.